x-pack/plugins/observability_solution/dataset_quality/common/data_streams_stats/integration.ts

```diff
@@ -5,22 +5,20 @@
  * 2.0.
  */
 
-import { DashboardType, IntegrationType } from './types';
+import { IntegrationType } from '../api_types';
 
 export class Integration {
   name: IntegrationType['name'];
   title: string;
   version: string;
   datasets: Record<string, string>;
   icons?: IntegrationType['icons'];
-  dashboards?: DashboardType[];
```

Comment:
Where are we storing the dashboards for the integration now? In the flyout state?

Reply:
We already discussed about this offline 👍🏼